server/igc/IgcAnalyzer.js

```diff
@@ -52,8 +53,16 @@ const IgcAnalyzer = {
 
     const igcAsPlainText = readIgcFile(flightDataObject);
 
-    //IGCParser needs lenient: true because some trackers (e.g. XCTrack) work with addional records in IGC-File which don't apply with IGCParser.
+    // IGCParser needs lenient: true because some trackers (e.g. XCTrack) work with addional records in IGC-File which don't apply with IGCParser.
     const igcAsJson = IGCParser.parse(igcAsPlainText, { lenient: true });
+
+    // Remove non flight fixes
+    const launchAndLanding = findLaunchAndLanding(igcAsJson);
```

Comment:
why is this an array?

Reply:
It can detect multiple flights in the same igc file.